### PR TITLE
规范代码，针对if判断后面只有一行的情况，也需要加大括号，不然以后修改容易发生错误

### PR DESCRIPTION
**Pull request by laddcn**
## What's the purpose of this PR

规范代码

## Brief changelog
规范代码，针对if判断后面只有一行的情况，也需要加大括号，不然以后修改容易发生错误

Follow this checklist to help us incorporate your contribution quickly and easily:

- [ ] Read the [Contributing Guide](https://github.com/ctripcorp/apollo/blob/master/CONTRIBUTING.md) before making this pull request.
- [ ] Write a pull request description that is detailed enough to understand what the pull request does, how, and why.
- [ ] Write necessary unit tests to verify the code.
- [ ] Run `mvn clean test` to make sure this pull request doesn't break anything.

### CODE REVIEW

File: apollo-configservice/src/main/java/com/ctrip/framework/apollo/configservice/util/InstanceConfigAuditUtil.java [hunk 1]
```diff
@@ -242,8 +242,12 @@ public Date getOfferTime() {
 
     @Override
     public boolean equals(Object o) {
```
Comment (Anilople) [inline apollo-configservice/src/main/java/com/ctrip/framework/apollo/configservice/util/InstanceConfigAuditUtil.java]: equals一般是由ide生成的，有时会不带大括号

Comment (laddcn) [reply apollo-configservice/src/main/java/com/ctrip/framework/apollo/configservice/util/InstanceConfigAuditUtil.java]: 是的，不过加上大括号还是好些，项目内的代码规范可以保持统一

**Comment by nobodyiam** [comment]
It seems this pr has conflicted changes with current master branch, please help to solve this conflict and commit again.

**Comment by laddcn** [comment]
I have resolved conflict with official master branch. @nobodyiam

**Pull request merged**